src/io/io_uring/SDL_asyncio_liburing.c

```diff
@@ -44,9 +44,17 @@ static bool (*AsyncIOFromFile)(const char *file, const char *mode, SDL_AsyncIO *
 // we never link directly to liburing.
 // (this says "-ffi" which sounds like a scripting language binding thing, but the non-ffi version
 // is static-inline code we can't lookup with dlsym. This is by design.)
-static const char *liburing_library = "liburing-ffi.so.2";
+#define SDL_DRIVER_LIBURING_DYNAMIC "liburing-ffi.so.2"
+static const char *liburing_library = SDL_DRIVER_LIBURING_DYNAMIC;
 static void *liburing_handle = NULL;
 
+SDL_ELF_NOTE_DLOPEN(
+    "io-io_uring",
+    "Support for async IO through liburing",
+    SDL_ELF_NOTE_DLOPEN_PRIORITY_REQUIRED,
```

Comment:
`RECOMMENDED` seems like plenty for this, or even `SUGGESTED`. (We ship without this in older branches of the Steam Runtime and nothing has complained...)

Reply:
This is suggested now.